src/views/IdentificationConfig.vue

```diff
@@ -13,7 +13,12 @@
             </li>
             <li>Durch manuelle Eingabe einer ID an einer openWB mit Display.</li>
           </ul>
-          Bei openWB Pro/Pro+ werden RFID-Tags nur bei angestecktem Fahrzeug erfasst!
+          Bei openWB Pro/Pro+ werden RFID-Tags nur bei angestecktem Fahrzeug erfasst!<br /><br />
+          Für die Erkennung kann auch ein Muster (mittels Wildcards) hinterlegt werden. Ein ? entspricht dabei
+          einem Zeichen, ein * beliebig vielen Zeichen. So ist bspw. die Erkennung von Fahrzeugen mit wechselnder ID
+          (u.a. Fahrzeuge der VW Gruppe) möglich. <br />
+          Es wird davon abgeraten die Funktion für die Entsperrung von öffentlich zugänglichen Ladepunkten zu nutzen,
+          um unbefugten Zugriff zu vermeiden.
```

Comment:
Bitte nicht mehrere Zeilenumbrüche für getrennte Absätze "missbrauchen". Dafür gibt es den Paragraphen `<p> </p>`.
```suggestion
          <p>Bei openWB Pro/Pro+ werden RFID-Tags nur bei angestecktem Fahrzeug erfasst!</p>
          <p>
            Für die Erkennung kann auch ein Muster (mittels Wildcards) hinterlegt werden. Ein ? entspricht dabei
            einem Zeichen, ein * beliebig vielen Zeichen. So ist bspw. die Erkennung von Fahrzeugen mit wechselnder ID
            (u.a. Fahrzeuge der VW Gruppe) möglich. <br />
            Es wird davon abgeraten die Funktion für die Entsperrung von öffentlich zugänglichen Ladepunkten zu nutzen,
            um unbefugten Zugriff zu vermeiden.
          </p>
```